README.md

```diff
@@ -0,0 +1,88 @@
+2. Set up a push integration in Iterable. This allows Iterable to communicate on your behalf with Apple's Push Notification Service
+
+If you haven't yet done so, you will need to enable push notifications for your application. This can be done by toggling `Push Notifications` under your target's `Capabilities` in Xcode. You can also do it directly in the app center on Apple's member center; go to `Identifiers -> App IDs -> select your app`. You should see `Push Notifications` under `Application Services`. Hit `Edit` and enable `Push Notifications`.
+
+You will also need to generate an SSL certificate and private key for use with the push service. See the links at the end of this section for more information on how to do that.
+
+Once you have your APNS certificates set up, go to `Integrations -> Mobile Push` in Iterable. When creating an integration, you will need to pick a name and a platform. The name is entirely up to you; it will be the `appName` when you use `registerToken` in our SDK. The platform can be `APNS` or `APNS_SANDBOX`; these correspond to the production and sandbox platforms. Your application will generate a different token depending on whether it is built using a development certificate or a distribution provisioning profile.
+
+![Creating an integration in Iterable](http://support.iterable.com/hc/en-us/article_attachments/202957719/Screen_Shot_2015-07-30_at_3.15.56_PM.png)
+
+For more information, see
+
+* [Configuring Push Notifications](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/AddingCapabilities/AddingCapabilities.html#//apple_ref/doc/uid/TP40012582-CH26-SW6)
+* [Creating Certificates](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/MaintainingCertificates/MaintainingCertificates.html#//apple_ref/doc/uid/TP40012582-CH31-SW32)
+* [Amazon's Guide to Creating Certificates](http://docs.aws.amazon.com/sns/latest/dg/mobile-push-apns.html)
+
+Congratulations, you've configured your mobile application to receive push notifications! Now, let's set up the Iterable SDK...
+
+# Automatic Installation (via CocoaPods)
+
+Iterable supports [CocoaPods](https://cocoapods.org) for easy installation. If you don't have it yet, you can install it with `Ruby` by running:
+```
+$ sudo gem install cocoapods 
+```
+
+To include the Iterable SDK in your project, you need to add it to your `Podfile`. If you don't have a `Podfile` yet, you can create one by running:
+```
+$ pod init
+```
+
+To add the Iterable pod to your target, edit the `Podfile` and include this line under the target:
+```
+pod 'Iterable-iOS-SDK'
+```
+
+Now, you need to tell Cocoapods to install the dependencies:
+```
+$ pod install
+```
+
+Congratulations! You have now imported the Iterable SDK into your project! 
+
+
+# Using the SDK
+1. Initialize the API with API key. The only required parameter is `apiKey`. You may also like to set default configuration parameters.
+
+	```
```

Comment:
Mark snippets with ```swift to enable syntax highlighting.

Reply:
This readme file is thoroughly incomplete. I am working on separate PR to work on documentation.